ucm2/Intel/sof-hda-dsp/HiFi.conf

```diff
@@ -8,7 +8,7 @@ SectionVerb {
 	Value.TQ "HiFi"
 }
 
-Include.hda-analog.File "/HDA/HiFi-analog.conf"
+Include.hda-analog.File "/Intel/sof-hda-dsp/HiFi-analog-sof.conf"
```

Comment:
you could drop the -sof postfix, the conf file is under sof-hda-dsp anyways?

How would the support for `sof-soundwire` going to be added?

Reply:
Yep, I will do it. I hope I can soon start also with SDW, it will be simpler than this thanks to endpoint dedicated DAI pipelines.

Reply:
so for SoundWire i'd really like to see no configurations in Enable, it should all be boot sequences.